parser/src/ast.rs

```diff
@@ -236,6 +236,7 @@ pub enum Expr<'a> {
     Tuple {
         elts: Vec<Spanned<Expr<'a>>>,
     },
+    Bool(bool),
```

Comment:
I'm unsure if this should be of type `&str` and then be mapped to `Expr::Bool("false")` and `Expr::Bool("true")`  instead. The current system works and we can map using a `literal_expression` but I *guess* that's just by coincidence because rust and Fe both use the same literals. If we would choose `yes` and `no`, then I think we would have to use strings.

Reply:
I'm fine with this the way it is.